package.json

```diff
@@ -46,7 +46,8 @@
     "test:protocol": "(cd packages/protocol && yarn run test)",
     "test:node": "(cd packages/node && yarn run test)",
     "test:node:watch": "(cd packages/node && yarn run test:watch)",
-    "test:node:watch:debug": "(cd packages/node && yarn run test:watch:debug)"
+    "test:node:watch:debug": "(cd packages/node && yarn run test:watch:debug)",
+    "cli:deployer": "lerna run --scope @airnode/deployer cli -- --"
```

Comment:
Small thing, but could we keep these ordered alphabetically please?

Reply:
Of course, I didn't realize that's the case :slightly_smiling_face:

Reply:
There are a few eslint package-json sorting plugins as well as a stand-alone script called sort-package-json